docs/tutorials/auth.md

```diff
@@ -415,3 +415,89 @@ scope by doing the following:
+
+### Configuration File
+
+As an example the following initial config can be used
+
+```yaml
+services:
+  - name: bundle-service
+    url: https://my-example-opa-bucket.s3.eu-central-1.amazonaws.com
+    credentials:
+      s3_signing:
+        environment_credentials: {}
+labels:
+  environment: production
+discovery:
+  name: discovery
+  prefix: "/applications/{{ .bundlename }}"
+```
+
+The variable `.bundlename` is the first argument in the following filters and can be in any format that OPA can understand, so for example application IDs from a registry, uuids, ...
+
+### Input Structures
+
+Input structures to policies follow those that are used by the [opa-envoy-plugin](https://github.com/open-policy-agent/opa-envoy-plugin), the existing [examples and documentation](https://www.openpolicyagent.org/docs/latest/envoy-primer/#example-input) apply also to Skipper. 
+
+### Passing context to the policy
+
+Generally there are two ways to pass context to a policy:
+
+1. as part of the labels on Open Policy Agent (configured in the configuration file, see below) that should be used for deployment level taxonomy,
+2. as part of so called context extensions that are part of the Envoy external auth specification. 
+
+This context can be passed as second argument to filters: 
+
+`authorizeWithRegoPolicy("my-app-id", "com.mycompany.myprop: myvalue")` 
+or `authorizeWithRegoPolicy("my-app-id", "{'com.mycompany.myprop': 'my value'}")` 
+
+The second argument is parsed as YAML, cannot be nested and values need to be strings. 
+
+In Rego this can be used like this `input.attributes.contextExtensions["com.mycompany.myprop"] == "my value"`
+
+### Quick Start Rego Playground
+
+A quick way without setting up Backend APIs is to use the [Rego Playground](https://play.openpolicyagent.org/).
+
+To get started pick from examples Envoy > Hello World. Click on "Publish" and note the random ID in the section "Run OPA with playground policy". 
+
+Place the following file in your local directory with the name `opaconfig.yaml`
+
+```yaml
+bundles:
+  play:
+    resource: bundles/{{ .bundlename }}
+    polling:
+      long_polling_timeout_seconds: 45
+services:
+  - name: play
+    url: https://play.openpolicyagent.org
+plugins:
+  envoy_ext_authz_grpc:
+    # This needs to match the package, defaulting to envoy/authz/allow
+    path: envoy/http/public/allow
+    dry-run: false
+decision_logs:
+  console: true
+```
+
+Start Skipper with 
+
+```
+skipper --enable-open-policy-agent --open-policy-agent-config-template opaconfig.yaml \
```

Comment:
I think we use falgs with a single dash in other examples, makes sense to have it consistent.

Reply:
Makes sense. Will change